--- fesapi-2.13.0.0/CMakeLists.txt	2025-07-04 18:09:02
+++ fesapi-2.13.0.0-modified/CMakeLists.txt	2026-01-23 14:59:41
@@ -82,7 +82,7 @@
 IF (NOT EXISTS ${ZLIB_INCLUDE_DIR})
 	MESSAGE(ERROR " The ZLIB_INCLUDE_DIR CMake variable is mandatory.")
 ENDIF ()
-FIND_PACKAGE (MINIZIP REQUIRED)
+FIND_PACKAGE (minizip CONFIG REQUIRED)
 
 # Szip (only useful when using static linking to HDF5)
 set (SZIP_LIBRARY_RELEASE SZIP_LIBRARY_RELEASE-NOTFOUND CACHE FILEPATH "Path to the file which contains the szip library release")
--- fesapi-2.13.0.0/cmake/FesapiCppConfig.cmake.in	2025-07-04 18:09:02
+++ fesapi-2.13.0.0-modified/cmake/FesapiCppConfig.cmake.in	2026-01-23 14:59:52
@@ -1,4 +1,9 @@
 include(CMakeFindDependencyMacro)
 
+find_dependency(HDF5 REQUIRED)
+find_dependency(minizip REQUIRED)
+find_dependency(ZLIB REQUIRED)
+find_dependency(Boost REQUIRED)
+
 # Include the actual targets
 include("${CMAKE_CURRENT_LIST_DIR}/FesapiCppTargets.cmake")
--- fesapi-2.13.0.0/src/CMakeLists.txt	2025-07-04 18:09:02
+++ fesapi-2.13.0.0-modified/src/CMakeLists.txt	2026-01-23 14:59:59
@@ -66,10 +66,10 @@
 endif()
 
 if (TARGET ZLIB::ZLIB)
-	target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE ZLIB::ZLIB MINIZIP::MINIZIP)
+	target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE ZLIB::ZLIB minizip::minizip)
 else ()
 	target_include_directories(${CPP_LIBRARY_NAME} SYSTEM PRIVATE ${ZLIB_INCLUDE_DIR})
-	target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE MINIZIP::MINIZIP)
+	target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE minizip::minizip)
 endif ()
 
 target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE Boost::boost)
