--- a/src/CMakeLists.txt	2026-01-22 16:15:05
+++ b/src/CMakeLists.txt	2026-01-22 16:15:05
@@ -9,7 +9,9 @@
 add_library (${CPP_LIBRARY_NAME} ${LIB_TYPE} "")
 
 # Create the version config header file in order for fesapi users to know the fesapi version at runtime.
-configure_file(${FESAPI_ROOT_DIR}/cmake/version_config.h.in ${FESAPI_ROOT_DIR}/src/version_config.h)
+if(NOT EXISTS ${FESAPI_ROOT_DIR}/src/version_config.h)
+	configure_file(${FESAPI_ROOT_DIR}/cmake/version_config.h.in ${FESAPI_ROOT_DIR}/src/version_config.h)
+endif()
 
 #configure the HDF5 hidt type to use in Fesapi according to the HDF5 version
 if (HDF5_VERSION VERSION_LESS "1.10")
@@ -26,7 +28,9 @@
 		target_compile_definitions(${CPP_LIBRARY_NAME} PRIVATE "HDF5_ROS3_ENABLED")
 	endif(HDF5_ROS3_ENABLED)
 endif ()
-configure_file(${FESAPI_ROOT_DIR}/cmake/HidtType.h.in ${FESAPI_ROOT_DIR}/src/common/HidtType.h)
+if(NOT EXISTS ${FESAPI_ROOT_DIR}/src/common/HidtType.h)
+	configure_file(${FESAPI_ROOT_DIR}/cmake/HidtType.h.in ${FESAPI_ROOT_DIR}/src/common/HidtType.h)
+endif()
 
 # Define the name of the directories where the various Energistics namespaces sources are located.
 set (RESQML_PREFIX_2 "resqml2")
@@ -138,7 +142,9 @@
 set (FESAPI_EML2_0_NS eml2_0 CACHE STRING "Namespace for eml2_0 objects")
 set (FESAPI_EML2_3_NS eml2_3 CACHE STRING "Namespace for eml2_3 objects")
 
-configure_file(${FESAPI_ROOT_DIR}/cmake/nsDefinitions.h ${FESAPI_ROOT_DIR}/src/nsDefinitions.h) # Overwrite if different
+if(NOT EXISTS ${FESAPI_ROOT_DIR}/src/nsDefinitions.h)
+	configure_file(${FESAPI_ROOT_DIR}/cmake/nsDefinitions.h ${FESAPI_ROOT_DIR}/src/nsDefinitions.h) # Overwrite if different
+endif()
 
 include(${CMAKE_CURRENT_SOURCE_DIR}/common/CMakeLists.txt)
 include(${CMAKE_CURRENT_SOURCE_DIR}/tools/CMakeLists.txt)
