diff --git a/CMakeLists.txt b/CMakeLists.txt
index 45f16b4..906f60e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,9 +42,6 @@ set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 # checking for required dependencies
 # ============================================================================
 
-# path to custom find_package modules
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules/")
-
 # hdf5
 set (WITH_LOCAL_HDF5 OFF CACHE BOOL "Force usage of a local and exotic version HDF5.")
 IF(WITH_LOCAL_HDF5)
@@ -82,7 +79,7 @@ FIND_PACKAGE (ZLIB)
 IF (NOT EXISTS ${ZLIB_INCLUDE_DIR})
 	MESSAGE(ERROR " The ZLIB_INCLUDE_DIR CMake variable is mandatory.")
 ENDIF ()
-FIND_PACKAGE (MINIZIP REQUIRED)
+FIND_PACKAGE (minizip CONFIG REQUIRED)
 
 # Szip (only useful when using static linking to HDF5)
 set (SZIP_LIBRARY_RELEASE SZIP_LIBRARY_RELEASE-NOTFOUND CACHE FILEPATH "Path to the file which contains the szip library release")
diff --git a/cmake/FesapiCppConfig.cmake.in b/cmake/FesapiCppConfig.cmake.in
index 1545c58..f208dba 100644
--- a/cmake/FesapiCppConfig.cmake.in
+++ b/cmake/FesapiCppConfig.cmake.in
@@ -1,4 +1,9 @@
 include(CMakeFindDependencyMacro)
 
+find_dependency(HDF5 REQUIRED)
+find_dependency(minizip REQUIRED)
+find_dependency(ZLIB REQUIRED)
+find_dependency(Boost REQUIRED)
+
 # Include the actual targets
 include("${CMAKE_CURRENT_LIST_DIR}/FesapiCppTargets.cmake")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index dffa132..3bbfe3b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -66,10 +66,10 @@ else()
 endif()
 
 if (TARGET ZLIB::ZLIB)
-	target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE ZLIB::ZLIB MINIZIP::MINIZIP)
+	target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE ZLIB::ZLIB minizip::minizip)
 else ()
 	target_include_directories(${CPP_LIBRARY_NAME} SYSTEM PRIVATE ${ZLIB_INCLUDE_DIR})
-	target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE MINIZIP::MINIZIP)
+	target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE minizip::minizip)
 endif ()
 
 target_link_libraries (${CPP_LIBRARY_NAME} PRIVATE Boost::boost)
@@ -195,8 +195,6 @@ list(APPEND ALL_SOURCES_AND_HEADERS
 
 target_sources(${CPP_LIBRARY_NAME} PRIVATE ${ALL_SOURCES_AND_HEADERS})
 
-target_include_directories(${CPP_LIBRARY_NAME} SYSTEM PRIVATE ${MINIZIP_INCLUDE_DIR})
-
 target_include_directories(${CPP_LIBRARY_NAME} INTERFACE
 	$<INSTALL_INTERFACE:include>
 )
@@ -370,7 +368,7 @@ else ()
 	)
 endif (NOT WITH_PYTHON_WRAPPING AND NOT WITH_DOTNET_WRAPPING AND NOT WITH_JAVA_WRAPPING)
 
-if (WIN32)
+if (WIN32 AND LIB_TYPE STREQUAL "SHARED")
 	#PDB for debug
 	INSTALL (
 		FILES $<TARGET_PDB_FILE:${CPP_LIBRARY_NAME}>